roles/metrics_server/defaults/main.yaml

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2025 VEXXHOST, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+metrics_server_helm_repository_name: metrics-server
+metrics_server_helm_repository_url: https://kubernetes-sigs.github.io/metrics-server
+metrics_server_helm_chart_name: metrics-server
+metrics_server_helm_chart_version: 3.12.2
```

Comment:
Can you add some rules into `renovate.json` to handle the automatic updates for this?

Also, I think if you want/can, we always use `*_helm_repository_url`, `*_helm_chart_name` and `*_helm_chart_version` -- perhaps we can create a generic `regex` handler for this pattern for all of them.

Reply:
Second approach sound very nice but renovate’s regexManagers can't natively link multiple variable across lines in a file. Renovate is magic until it's not;/